lib/ansible/modules/database/elasticsearch/elasticsearch_snapshot.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python3
+# elasticsearch_snapshot: Do snapshots of elasticsearch indices
+#
+# Copyright (C) 2017 Lyz <lyz@riseup.net>
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: elasticsearch_snapshot
+short_description: Do snapshots of elasticsearch indices
+description: Do snapshots of elasticsearch indices
+version_added: '2.5'
+requirements:
+  - "python >= 3.0"
+  - requests
```

Comment:
Fixed in ee1c8669c 

Although they do [support python3 for modules](http://docs.ansible.com/ansible/latest/dev_guide/developing_python3.html)